website/yarn.lock

```diff
@@ -9,7 +9,7 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4", "@babel/code-frame@^7.8.3":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4":
```

Comment:
It's pretty odd that this changed. I would expect only `he` to be added to the lockfile, with no other changes. Does running `yarn` on master (without this PR) produce these extra changes as well?

Reply:
Yes. I have just pulled the updated master and then added `he`.

Reply:
@motiz88 I have added `he` as devDependency in the latest commit https://github.com/facebook/react-native-website/pull/2143/commits/547db4e82f649f42181c74d199f9a2aec9688135.

Reply:
Hey @ani4aniket, have you had a chance to check whether running `yarn` on master _before this PR_ produces changes to `yarn.lock`? If so, those should be fixed in a separate commit prior to this PR.

Reply:
Sure, will do that.

Reply:
@motiz88 Yes running `yarn` on master is producing the changes to `yarn.lock`. I've tested this on the updated master.

![Screenshot 2020-08-20 at 2 51 52 PM](https://user-images.githubusercontent.com/31184591/90752354-e62fa080-e2f4-11ea-8bf9-51b158f0d9b9.png)

Reply:
@motiz88 This `yarn.lock` change is fine, it's the last know to me side effect of the merge of earlier auto-gen related PRs (which was not fixed yet).